_docs/operations/encryption.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: guide
+title: StorageOS Docs - Encryption
+anchor: operations
+module: operations/encryption
```

Comment:
Please, lets follow uniformity between the title in the page and the title in the guides. Otherwise the menu sends you to a page with a different title. 

I don't have a preference for one or the other, but lets use the same. 

`encryption` or `Encrypted Volumes`